app/Subscriber.php

```diff
@@ -45,7 +45,7 @@ public function messages()
 
     public function tags()
     {
-        return $this->belongsToMany('App\Tag');
+        return $this->belongsToMany(Tag::class);
```

Comment:
Does this enable the `$subscriber->tags->pluck('id')`? Or is it just a convention to use the class reference instead of a string?

Reply:
Yeah it’s just a class name convention, the advantage over a normal string is that an IDE can recognize it so you get “go to class” functionality.